dallinger/frontend/templates/dashboard_develop.html

```diff
@@ -0,0 +1,49 @@
+{% extends "base/dashboard.html" %}
```

Comment:
I like the idea of being able to initiate a `launch` from the dashboard, though it might be nice if the run script could optionally do it (e.g. run flask in the background, run `launch`, then bring flask back to foreground). Unfortunately, I couldn't get the dashboard to work at all. The `run.sh` command doesn't print a dashboard password and the `browser` command doesn't set the password, so I got an error when trying to launch the dashboard route or access it. Setting a `DASHBOARD_PASSWORD` environment variable didn't seem to help.

Reply:
I had `dashboard_user` and `dashboard_password` set in `~/.dallingerconfig`, which is why I never ran into this. I updated the `dallinger develop bootstrap` command to write these values to the config.txt file that gets put in the develop directory, so this should now work even if you don't have those values set.